(Google Play version) Do not crash when Google Play services aren't available

### Checklist

- [x] I have used the search function for [**OPEN**](https://github.com/AntennaPod/AntennaPod/issues) issues to see if someone else has already submitted the same bug report.
- [x] I have **also** used the search function for [**CLOSED**](https://github.com/AntennaPod/AntennaPod/issues?q=is%3Aissue+is%3Aclosed) issues to see if the problem is already solved and just waiting to be released.
- [x] I will describe the problem with as much detail as possible.
- [x] If the bug only to occurs with a certain podcast, I will include the URL of that podcast.

### App version

3.8.0

### Android version

15

### Device model

Oneplus

### First occurred

Unknown

### Steps to reproduce

1. Get the Google Play version while Google Play services are disabled.
2. Crash.

### Expected behaviour

Not crash.

### Current behaviour

Crash.

### Logs

_No response_

Not sure why you weren't able to provide the logs @jwkwshjsjsj , but I can replicate the crash on start when disabling Google Play Services:

## Environment
Android version: 15
OS version: 5.4.274-qgki-gca7127692058
AntennaPod version: 3.9.0
Model: FP5
Device: FP5
Product: FP5

## Crash info
Time: 14-09-2025 23:10:08
AntennaPod version: 3.9.0

## StackTrace
```
java.lang.RuntimeException: Unable to create application de.danoeh.antennapod.PodcastApp: java.lang.IllegalArgumentException: de.danoeh.antennapod: Targeting U+ (version 34 and above) disallows creating or retrieving a PendingIntent with FLAG_MUTABLE, an implicit Intent within and without FLAG_NO_CREATE and FLAG_ALLOW_UNSAFE_IMPLICIT_INTENT for security reasons. To retrieve an already existing PendingIntent, use FLAG_NO_CREATE, however, to create a new PendingIntent with an implicit Intent use FLAG_IMMUTABLE.
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7529)
	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2417)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loopOnce(Looper.java:232)
	at android.os.Looper.loop(Looper.java:317)
	at android.app.ActivityThread.main(ActivityThread.java:8751)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:892)
Caused by: java.lang.IllegalArgumentException: de.danoeh.antennapod: Targeting U+ (version 34 and above) disallows creating or retrieving a PendingIntent with FLAG_MUTABLE, an implicit Intent within and without FLAG_NO_CREATE and FLAG_ALLOW_UNSAFE_IMPLICIT_INTENT for security reasons. To retrieve an already existing PendingIntent, use FLAG_NO_CREATE, however, to create a new PendingIntent with an implicit Intent use FLAG_IMMUTABLE.
	at android.os.Parcel.createExceptionOrNull(Parcel.java:3246)
	at android.os.Parcel.createException(Parcel.java:3226)
	at android.os.Parcel.readException(Parcel.java:3209)
	at android.os.Parcel.readException(Parcel.java:3151)
	at android.app.IActivityManager$Stub$Proxy.getIntentSenderWithFeature(IActivityManager.java:7015)
	at android.app.PendingIntent.getActivityAsUser(PendingIntent.java:581)
	at android.app.PendingIntent.getActivity(PendingIntent.java:562)
	at android.app.PendingIntent.getActivity(PendingIntent.java:526)
	at com.google.android.gms.internal.common.zzd.zza(SourceFile:1)
	at com.google.android.gms.common.GoogleApiAvailabilityLight.getErrorResolutionPendingIntent(SourceFile:3)
	at com.google.android.gms.common.GoogleApiAvailability.showErrorNotification(SourceFile:1)
	at de.danoeh.antennapod.net.ssl.SslProviderInstaller.install(SourceFile:15)
	at de.danoeh.antennapod.ClientConfigurator.initialize(SourceFile:48)
	at de.danoeh.antennapod.PodcastApp.onCreate(SourceFile:45)
	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1387)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7523)
	... 9 more
Caused by: android.os.RemoteException: Remote stack trace:
	at com.android.server.am.ActivityManagerService.getIntentSenderWithFeatureAsApp(ActivityManagerService.java:5689)
	at com.android.server.am.ActivityManagerService.getIntentSenderWithFeature(ActivityManagerService.java:5631)
	at android.app.IActivityManager$Stub.onTransact(IActivityManager.java:3468)
	at com.android.server.am.ActivityManagerService.onTransact(ActivityManagerService.java:2844)
	at android.os.Binder.execTransactInternal(Binder.java:1500)

```

--- a/net/ssl/src/play/java/de/danoeh/antennapod/net/ssl/SslProviderInstaller.java
+++ b/net/ssl/src/play/java/de/danoeh/antennapod/net/ssl/SslProviderInstaller.java
@@ -2,7 +2,6 @@ package de.danoeh.antennapod.net.ssl;
 
 import android.content.Context;
 import com.google.android.gms.common.GoogleApiAvailability;
-import com.google.android.gms.common.GooglePlayServicesNotAvailableException;
 import com.google.android.gms.common.GooglePlayServicesRepairableException;
 import com.google.android.gms.security.ProviderInstaller;
 
@@ -13,7 +12,7 @@ public class SslProviderInstaller {
         } catch (GooglePlayServicesRepairableException e) {
             e.printStackTrace();
             GoogleApiAvailability.getInstance().showErrorNotification(context, e.getConnectionStatusCode());
-        } catch (GooglePlayServicesNotAvailableException e) {
+        } catch (Exception e) {
             e.printStackTrace();
         }
     }

